packages/custom-functions-metadata/test/cases/error-unsupported-return-type-cellvalue/functions.ts

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+/**
+ * Cell Value types are not supported as return types
+ * @customfunction
+ */
+function testUnsupportedCellValue(x): Excel.BooleanCellValue {
```

Comment:
As per my other comment, I think this can/should be relaxed. The function can return a cell value or error cell value type. For the linked entity provider function, it MUST return Promise<Excel.CellValue> since it needs to return the linked entity.

Reply:
See my other comment. Dev should continue to use Promise<any> at this stage until we actually support return type.

Reply:
Metadata can contain boolean/string/number as return types. Excel used to do type check on these but it no longer does for some reason.

That said, we had some discussions, and we agree that converting CellValues as return types to "any" is better. I'll update the PR.